vendor.conf

```diff
@@ -27,7 +28,7 @@ github.com/opencontainers/runc master
 github.com/opencontainers/image-spec v1.0.0
 # -- start OCI image validation requirements.
 github.com/opencontainers/runtime-spec v1.0.0
-github.com/opencontainers/image-tools 6d941547fa1df31900990b3fb47ec2468c9c6469
+github.com/opencontainers/image-tools da84dc9dddc823a32f543e60323f841d12429c51
```

Comment:
This reverts image-tools to an older commit, notably using upper-case `Sirupsen`.  But this commit does not actually update the vendored package?!